docs/zh/connector-v2/formats/cdc-compatible-debezium-json.md

```diff
@@ -46,6 +46,7 @@ sink {
     plugin_input = "table1"
 
     bootstrap.servers = "localhost:9092"
+    topic = "${topic}"
```

Comment:
Can we tell users the topic is optional not required?

Reply:
Keep it optional only for compatibility with older versions